charts/zabbix/templates/service.yaml

```diff
@@ -109,7 +104,11 @@ spec:
       nodePort: {{ .Values.zabbixAgent.service.nodePort }}
       {{- end }}
   selector:
-    app: {{ template "zabbix.fullname" . }}-zabbix-agent
+    {{- include "zabbix.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: server
+    {{- if .Values.zabbixServer.zabbixServerHA.enabled }}
+    {{ .Values.zabbixServer.zabbixServerHA.haLabelsSidecar.labelName }}: "active"
+    {{- end }}
```

Comment:
Hi @fibbs 

If someone installs only zabbix-agent using daemonset mode without installing zabbix-server or HA, they may encounter errors here. Right?

What do you think about adding an if conditional to check if zabbix-server or HA is enabled before adding these labels?

Reply:
Oops, yes, you are right again. This slipped in because I was supposing the service I have re-added was actually for Agent in sidecar mode, which it isn't. So the selector to go to the zabbix server pods is wrong altogether and should also be reverted to match the global selector labels plus `app.kubernetes.io/component: agent`.
Commit just pushed.

Reply:
Great team work. Thanks @fibbs